reviews/api/views.py

```diff
@@ -405,6 +405,38 @@ def get_queryset(self):
         return [value for key, value in reviews.items()]
 
 
+class InRevisionApiView(BaseReviewApiView):
+    """Return reviews for proposals which have been sent back for revision,
+    but for which a revision has not yet been submitted"""
+
+    default_sort = ('date_start', 'desc')
+    group_required = [settings.GROUP_SECRETARY]
+
+    def get_queryset(self):
+        # 1. Find reviews of revisions:
+        # A proposal not having one of these means
+        # that the review of its parent is "in revision"
+        revision_reviews = Review.objects.filter(
+            proposal__is_revision=True,    # Not a copy
+            stage__gte=Review.ASSIGNMENT,  # Not a supervisor review
+        )
+        # 2. Get candidate reviews:
+        # All reviews whose conclusion is "revision necessary"
+        candidates = Review.objects.filter(
+            stage=Review.CLOSED,
```

Comment:
I'm pretty sure you forgot to filter by commission?

Reply:
Yes I completely forgot